runner.py

```diff
@@ -85,6 +85,18 @@ def parallel_process(processing_dir, fastq_files, reference_file, quality_thresh
         raise Exception("Processing failed. This may be due to RAM overload."
                         "To avoid this try running with --max_memory max_available_RAM_in_MB")
 
+    path = os.path.join(processing_dir, 'Exceptions_during_process_fastq.txt')
+    if os.path.exists(path):
+        file_exception = open(path, 'r')
+        content = file_exception.read()
+        file_exception.close()
+        exception_printed = "There were exception in the parallel processing. see " \
+                            "processing\Exceptions_during_run_blast.txt. "
```

Comment:
an exception could also happen in basecall so it is not specific to blast

Reply:
added to be more informative- an exception will be raised also in this case.